src/runtime_src/core/tools/common/TestRunner.cpp

```diff
@@ -391,6 +391,11 @@ std::string
 TestRunner::findPlatformPath(const std::shared_ptr<xrt_core::device>& _dev,
                  boost::property_tree::ptree& _ptTest)
 {
+#ifdef _WIN32
+  boost::ignore_unused(_dev);
+  boost::ignore_unused(_ptTest);
+  return xrt_core::find_platform_path();
```

Comment:
We should try and not have any ifdefs in higher level code. We should reexamine how we find xclbins and their paths.

Personally, I think the platform path is something a device should define not a system. Linux will also support IPU one day which will break the current changes. Better to sort this issue out now rather than patching a hole.